api/test/e2e/route_test.go

```diff
@@ -0,0 +1,233 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package e2e
+
+import (
+	"net/http"
+	"testing"
+)
+
+func TestRoute_Host_Params(t *testing.T) {
```

Comment:
how about these steps? it seems clear:

1. hit route `/hello` -> will fail, no route
2. add route, `/hello` -> success
3. hit route `/hello` -> success
4. hit route `/not_found` ->will fail, not match route
5. delete route `/hello` -> success
6. hit route `/hello` -> will fail, no route

Reply:
I had  split them into `TestRoute_Create_With_Hosts`, `TestRoute_Update_Routes_With_Hosts`,  `TestRoute_Delete_Routes_With_Hosts`